crypto-common/src/lib.rs

```diff
@@ -59,12 +59,12 @@ impl<T: BlockSizeUser> BlockSizeUser for &mut T {
 }
 
 /// Trait implemented for supported block sizes, i.e. for types from `U1` to `U255`.
-pub trait BlockSizes: ArrayLength<u8> + sealed::BlockSizes + 'static {}
+pub trait BlockSizes: ArraySize + sealed::BlockSizes + 'static {}
```

Comment:
I wonder if these `'static` bounds are still needed. I think we introduced them to work around an old issue which may have been already fixed. But I guess we can leave them be.

Reply:
Good catch! We don't because `ArraySize` already bounds on `'static`: https://docs.rs/hybrid-array/0.2.0-pre.5/hybrid_array/trait.ArraySize.html

Reply:
Duplicate `'static` bounds removed in c3ede80